src/components/seekbar/seekbar.js

```diff
@@ -424,7 +424,7 @@ class SeekBar extends Component {
       this.props.duration *
       ((xPosition - this._seekBarElement.offsetLeft - this.getOffset(this.props.playerElement).left) / this._seekBarElement.clientWidth);
     time = parseFloat(time.toFixed(2));
-    if (time < 0) return 0;
+    if (time < this.getPlayerTimeOffset()) return this.getPlayerTimeOffset();
```

Comment:
`const playerTimeOffset = this.getPlayerTimeOffset()` instead of calling this.getPlayerTimeOffset() twice
also please change the short if to the regular syntax